libcudacxx/test/libcudacxx/libcxx/text/format/format.formatter/format.formatter.spec/parsed_spec_abi.pass.cpp

```diff
@@ -0,0 +1,85 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// <cuda/std/format>
+
+// cuda::std::__fmt_parsed_spec
+
+#include <cuda/std/__format_>
+#include <cuda/std/cassert>
+#include <cuda/std/cstddef>
+#include <cuda/std/cstring>
+#include <cuda/std/utility>
+
+template <class T>
+__host__ __device__ cuda::std::__fmt_parsed_spec<T> get_test_parsed_spec() noexcept
```

Comment:
I would rather have this be a function that takes a `__fmt_parsed_spec` be reference and then verifies the desired values.

We can then call this function from host and then within the kernel